backend/Recodo.BLL/Services/EmailService.cs

```diff
@@ -0,0 +1,34 @@
+﻿using MailKit.Net.Smtp;
+using Microsoft.Extensions.Configuration;
+using MimeKit;
+using System;
+using System.Threading.Tasks;
+
+namespace Thread_.NET.BLL.Services
+{
+    public class EmailService
+    {
+        public static async Task SendEmailAsync(string email, string subject, string message, IConfiguration configuration)
+        {
+            var emailMessage = new MimeMessage();
+
+            emailMessage.From.Add(new MailboxAddress("admin@recodo.com", "admin@recodo.com"));
+            emailMessage.To.Add(new MailboxAddress(email, email));
+            emailMessage.Subject = subject;
+            emailMessage.Body = new TextPart(MimeKit.Text.TextFormat.Text)
+            {
+                Text = message
+            };
+
+            using (var client = new SmtpClient())
```

Comment:
I'm not really happy about using SmtpClient, users will often get such email in spam...

Reply:
Migrated to SendGrid